### ISSUE TITLE
Ability to control the 'drone' object and fly it around. 

### ISSUE THREAD

**Issue by austinchambers** [austinchambers/multiplayer-vr#10]
This issue covers what happens after the drone is possessed. You need to be able to fly it around, with all the flight mechanics that entails.

**Comment by damlaren** [comment]
I guess I'll work on the drone, and hopefully that will be similar to moving the sub?

**Comment by damlaren** [comment]
Have helo moving up and down now; not committing code yet. I'm liking this blueprint system.

**Comment by damlaren** [comment]
Still need to commit, but a few notes on what I have:

Controls:
Z/C to rotate possessed helicopter
Q/E to move up and down.
A/D to move side-side.
W/S to move forward & back. There is no constant forward motion right now but that's easy to change. We'll want a more complex movement model with acceleration & deceleration later.

I was able to move the helos by calling SetActorLocation, but "Set Actor Rotation" didn't work for rotating them. I had to call "Add Controller Yaw Input" and I don't know why that is.
